### ISSUE TITLE
Stackable Shadows

### ISSUE THREAD

**Issue by Deanphillips** [soflyy/oxygen-bugs-and-features#636]
I'd love to see the ability to stack shadows. 

This is possible with both webflow and thrive. 

This also goes well with the idea of layering like the stackable background suggestion that both have too 

https://github.com/soflyy/oxygen-bugs-and-features/issues/22

**Comment by krstivoja** [comment]
+1

**Comment by wplit** [comment]
I agree, you can get much more smoother / realistic shadows by stacking them

see [this article](https://tobiasahlin.com/blog/layered-smooth-box-shadows/) for inspiration.

Regarding the stacking of CSS rules, this could be applied more generically couldn't it? So any CSS rule that allows multiple values.

Backgrounds (incl gradients), box-shadows, font-family (for font stack), transitions, etc

**Comment by bzle** [comment]
+1
